silkrpc/core/evm_debug.hpp

```diff
@@ -121,7 +121,16 @@ struct DebugTrace {
     DebugConfig debug_config;
 };
 
+struct DebugTraceResult {
+    DebugTrace debug_trace;
+};
```

Comment:
This will be removed using the new approach.

Reply:
ok